### DISCUSSION TITLE
Anyone selling a boards, kits or complete units

### DISCUSSION THREAD

**Discussion by Schalburg** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#573] [General]
Loking for a ready made set or a compleat kit

**Comment by No-World-Order** [comment]
+1 on this, If anyone in Germany / France / Belgium or Netherlands can create one i would be happy to pay for it! :)

**Reply by steffzbl** [reply]
i've got them in the Netherlands

**Reply by No-World-Order** [reply]
Nice! Can you sent me a DM or mail rick @ rsloot. nl? I have a board, but it's not working because the cable soldering was wrong. I'm in Rotterdam area btw.

**Comment by Schoxian** [comment]
I have 3 kits, with red llc, nodemcuv3, and header pins for you to solder on, along with v2b boards in US.  You will have to assemble and get the wires and connectors, can be had on amazon.  You can contact me at rob@schoxtopia.me to discuss.   Thanks.

**Comment by reevery** [comment]
Given most places only print minimum 5 circuit boards, we could have a wiki-like thread where we can share/sell our left-overs. @Rob-Schox I will get in touch, and I can also bring back to Europe for @No-World-Order if still needed. Otherwise I will arrange a print of 5 and will then have 4 left.

**Comment by No-World-Order** [comment]
I still need it indeed. Sent an e-mail, but no response yet :).

**Reply by metaljay** [reply]
https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/discussions/580

**Comment by LateWiksi** [comment]
Purchased just a couple of weeks ago ready chip from Ebay (it came from Austria). Setting it up was quite easy. I did attach the Wi-Fi module circuit to general electric circuit box and used hot glue to seal the holes that I cut for wires.

Same guy from Austria still seems to have these boards or he's manufacturing them more:
https://www.ebay.com/itm/115870120599?hash=item1afa65ba97:g:2lsAAOSwzcpiaBFx

Just letting you know there are these boards available. Delivery to Finland took around 1,5 weeks (because of post companies, not because of seller) and it came as normal mail to our mailbox.

**Reply by metaljay** [reply]
Wow that’s silly money!

**Comment by metaljay** [comment]
Selling some here: https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/discussions/580

**Comment by Gifford47** [comment]
I'm also selling 4 bare PCBs. All untouched without any components.
Dispatch from Germany.

**Reply by Chris-lal** [reply]
Hi Gifford47, I'm interested. Mind sending one to France?

**Reply by Gifford47** [reply]
@Chris-lal yes, no problem. write me an email to: paors897proton.me@proton.me

**Reply by Chris-lal** [reply]
Thanks but I got one in the mean time

**Comment by ashb** [comment]
If anyone wants a V2 PCB (not 2b), Can include Red TTL convertor and esp8266, and 4 wire connectors. Contact me: ash.berlin@gmail.com. UK based, happy to ship where ever but you pay postage :)
